shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/helpers/extensions/String+Regex.kt

```diff
@@ -0,0 +1,9 @@
+@file:Suppress("MatchingDeclarationName")
+package co.nimblehq.blisskmmic.helpers.extensions
+
+fun String.isValidEmail(): Boolean {
+
+    val expression = "\\b[\\w\\.-]+@[\\w\\.-]+\\.\\w{2,4}\\b"
+    val regex = Regex(expression)
+    return this.isNotEmpty() && this.matches(regex)
+}
```

Comment:
Some cleanup suggestions 🙏
```suggestion
fun String.isValidEmail(): Boolean {
    val expression = "\\b[\\w\\.-]+@[\\w\\.-]+\\.\\w{2,4}\\b"
    val regex = Regex(expression)
    return isNotEmpty() && matches(regex)
}
```